tools/azure-sdk-tools/devtools_testutils/storage_testcase.py

```diff
@@ -56,11 +56,13 @@ def create_resource(self, name, **kwargs):
                 name,
                 {
                     'sku': {'name': self.sku},
-                    'location': self.location,
+                    'location': "canadacentral",
```

Comment:
curious - why are we doing this?

Reply:
don't worry I'm trying to run live test, this is just for test